KVMarketSquare/Favorites.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Favorites.swift
+//  KVMarketSquare
+//
+//  Created by Ka Hin Ng on 2021-12-13.
+//
+
+import Foundation
+
+class AppData: ObservableObject {
+    // this should be of type some link to a shop identifier
+    @Published var favoriteShops: [String] = ["hi", "bye"]
```

Comment:
{
  "user_sites": [
    {
      "user_id": "132061561",
      "site_id": "301174549463861952"
    },


this is how the lookup POST for each seller url looks - now idk if a single user id can have multiple siteids and what that would look like here but 🤷   

so i anticipate a seller shop being represented by a userId and siteId.  I do not think I would want to locally save urls b/c sellers can change those pretty quickly if they wanted I think